scipy/stats/tests/test_multivariate.py

```diff
@@ -1150,6 +1155,93 @@ def test_haar(self):
         ks_tests = [ks_2samp(proj[p0], proj[p1])[1] for (p0, p1) in pairs]
         assert_array_less([ks_prob]*len(pairs), ks_tests)
 
+class TestRandomCorrelation(TestCase):
+    def test_reproducibility(self):
+        np.random.seed(514)
+        eigs = (.5, .8, 1.2, 1.5)
+        x = random_correlation.rvs((.5, .8, 1.2, 1.5))
+        expected = np.array([[1., -0.20387311, 0.18366501, -0.04953711],
+                             [-0.20387311, 1., -0.24351129, 0.06703474],
+                             [0.18366501, -0.24351129, 1., 0.38530195],
+                             [-0.04953711, 0.06703474, 0.38530195, 1.]])
+        assert_array_almost_equal(x, expected)
+
```

Comment:
Yes. I was assuming you had added the test to RandomCorrelation actually, but forgot to check before pushing. I have added tests so that all 3 of Ortho, SpecialOrtho, and RandomCorrelation have these checks. This ENH patch now also updates other tests and doesn't just add a new feature; wasn't sure whether it was better to split into two patches.